operations/iuf/workflows/management_rollout.md

```diff
@@ -259,6 +259,8 @@ Refer to that table and any corresponding product documents before continuing to
      - All management NCNs have been upgraded to the image and CFS configuration created in the previous steps of this workflow
      - Per-stage product hooks have executed for the `management-nodes-rollout` stage
 
+    > **`NOTE`** After `management-nodes-rollout` stage for management NCNs is completed, re-initialize cray CLI. Refer to [Configure the Cray Command Line Interface (cray CLI)](../../configure_cray_cli.md)
```

Comment:
This should be moved up to line 256. It would be very easy to miss if it were left here.

Reply:
Good point @leliasen-hpe.    I think immediately after the ncn-m001 upgrade would be a better spot.

Reply:
Thanks for suggestion @leliasen-hpe

Reply:
Hi @leliasen-hpe
I have done the required changes in docs .

Reply:
Thanks @Srinivas-Anand-HPE that looks better to me!